### ISSUE TITLE
error when compiling

### ISSUE THREAD

**Issue by zzzlxhhh** [SuperScientificSoftwareLaboratory/TileSpGEMM#1]
Hi Niu!
You've done a great work in SpGEMM using Tile strategy . While I want to do some experiment in my own environment, I met with some error after 'make'. I'm testing on a NVIDIA GeForce RTX 2080 Ti with compute capacity of  7.5 in unbuntu20.04  with cuda-11.4 .
error output:
```
@Server:~/TileSpGEMM/src$ make
nvcc -O3 -w -arch=compute_61 -code=sm_75 -gencode=arch=compute_61,code=sm_75 -Xcompiler -fopenmp -Xcompiler -mfma main.cu -o test -I/home/zhanglx/cuda-11.4/include -L/home/zhanglx/cuda-11.4/lib64  -lcudart  -lcusparse   -D VALUE_TYPE=double
/usr/lib/gcc/x86_64-linux-gnu/9/include/avx512fintrin.h(9146): error: identifier "__builtin_ia32_rndscaless_round" is undefined

/usr/lib/gcc/x86_64-linux-gnu/9/include/avx512fintrin.h(9155): error: identifier "__builtin_ia32_rndscalesd_round" is undefined

/usr/lib/gcc/x86_64-linux-gnu/9/include/avx512fintrin.h(14797): error: identifier "__builtin_ia32_rndscaless_round" is undefined

/usr/lib/gcc/x86_64-linux-gnu/9/include/avx512fintrin.h(14806): error: identifier "__builtin_ia32_rndscalesd_round" is undefined

/usr/lib/gcc/x86_64-linux-gnu/9/include/avx512dqintrin.h(1365): error: identifier "__builtin_ia32_fpclassss" is undefined

/usr/lib/gcc/x86_64-linux-gnu/9/include/avx512dqintrin.h(1372): error: identifier "__builtin_ia32_fpclasssd" is undefined

6 errors detected in the compilation of "main.cu".
make: *** [Makefile:29: make] Error 1
```
1、maybe it's because of the compute capacity not matching with yours. 2、maybe it's because of the gcc --version not matching.
It would be highly appreciated if you can provide further information of your environment or give me some suggestion to solve the compilation error

**Comment by zzzlxhhh** [comment]
well , I solved the problem by install the same gcc&g++7.5.0  version mentioned in paper . My unbuntu is 22.04 whose default gcc&g++ version is 9.4.0 , so it's a trivial issue easy to be ignored. here is a link to show the way to install multiple versions of gcc&g++ on your system.[How to install multiple versions of GCC and G++ on Ubuntu](https://www.fosslinux.com/39386/how-to-install-multiple-versions-of-gcc-and-g-on-ubuntu-20-04.htm)

**Comment by VeritasFutureKF** [comment]
do you solve the problem at last?
